.github/workflows/test.yml

```diff
@@ -0,0 +1,106 @@
+name: "Test aws-robomaker-github-actions"
+
+on:
+  push:
+  pull_request:
+    branches:
+      - master
+      - 'releases/*'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  test_robomaker-sample-app-ci:
+    runs-on: ubuntu-latest
+    
+    strategy:      
+      matrix:
+        distro: ['kinetic', 'melodic']
+        gazebo: [7, 9]
+        include:
```

Comment:
Remember to test ROS2 Dashing as well (add a new workflow YAML file, like `test_ros2.yml`).

Reply:
I thought we discussed before we want to run this for ros 1 only.

Reply:
Okay, let's add ROS2 Dashing as a test in a separate. We should do it at some point.